### PR TITLE
refactor: return type of toChars() changed to const

### PR DESCRIPTION
**Pull request by WalterBright**
good step forward

### CODE REVIEW

File: src/attrib.d [hunk 1]
```diff
@@ -453,9 +453,9 @@ public:
         return createNewScope(sc, sc.stc, this.linkage, sc.protection, sc.explicitProtection, sc.structalign, sc.inlining);
     }
 
-    override char* toChars() const
+    override const(char)* toChars() const
     {
-        return cast(char*)"extern ()";
+        return "extern ()".ptr;
```
Comment (yebblies) [inline src/attrib.d]: `.ptr` is not necessary on many of these.

**Comment by yebblies** [comment]
Apart from that looks good.

**Comment by WalterBright** [comment]
> Apart from that looks good.

Turn on automerge, then?

**Comment by yebblies** [comment]
Auto-merge toggled on

**Comment by WalterBright** [comment]
thanks!

**Pull request merged**